tests/system/test_pandas.py

```diff
@@ -283,15 +283,6 @@ def test_load_table_from_dataframe_w_nulls(bigquery_client, dataset_id):
         bigquery.SchemaField("ts_col", "TIMESTAMP"),
     )
 
-    table_schema = scalars_schema + (
-        # TODO: Array columns can't be read due to NULLABLE versus REPEATED
-        #       mode mismatch. See:
-        #       https://issuetracker.google.com/133415569#comment3
-        # bigquery.SchemaField("array_col", "INTEGER", mode="REPEATED"),
-        # TODO: Support writing StructArrays to Parquet. See:
-        #       https://jira.apache.org/jira/browse/ARROW-2587
-        # bigquery.SchemaField("struct_col", "RECORD", fields=scalars_schema),
-    )
```

Comment:
Are we just bailing on those two `TODO` entries?

Reply:
+1, is this now off-plan?

Reply:
These tests are actually already covered in the v3 branch. https://github.com/googleapis/python-bigquery/blob/3b3ebff56004e061f58ecdd06fc9020423b890e2/tests/system/test_pandas.py#L294